hyperstart/api/grpc/oci.proto

```diff
@@ -0,0 +1,456 @@
+syntax = "proto3";
+
+package grpc;
+
+message Spec {
+	// Version of the Open Container Initiative Runtime Specification with which the bundle complies.
+	string Version = 1;
+
+	// Process configures the container process.
+	OCIProcess Process = 2;
+
+	// Root configures the container's root filesystem.
+	Root Root = 3;
```

Comment:
Shouldn't this (and all the other optional entries like `ConsoleSize`, `Mounts`, ...) be annotated with `optional` to comply as closely as possible with the OCI spec?

Reply:
@jodh-intel The latest protobuf version (v3) no longer supports `optional`:
https://developers.google.com/protocol-buffers/docs/reference/go-generated#fields

That's unfortunate, but seems it's been a long debated design decision among the protobuf community members.

Reply:
Ah - thanks. I wasn't aware of that.